sources/zsh/aliases.d/volatility3

```diff
@@ -0,0 +1 @@
+alias volatility3='/root/.local/pipx/venvs/volatility3/bin/vol'
```

Comment:
I'm not sure this alias is needed. I think it could be changed like this: `alias volatility3=vol`

Reply:
I put this alias because `vol` is already used by `volatility2`

Reply:
Or use `vol3` for `volatility3` and keep `vol` for `volatility2` ?

Reply:
I don't know. It would be clearer to put 'vol2' and 'vol3' ?